src/main/java/org/datadog/jmxfetch/util/LogLevel.java

```diff
@@ -0,0 +1,121 @@
+package org.datadog.jmxfetch.util;
+
+import java.util.logging.Level;
+
+/**
+ * LogLevel used for internal logging to match Datadog Agent levels.
+ * Comparison table with java.util.logging:
+ * <pre>
+ *  JUL     |  JMXFetch LogLevel
+ * ----------------------------
+ *  ALL     |  ALL
+ *  SEVERE  |  ERROR
+ *  WARNING |  WARN
+ *  INFO    |  INFO
+ *  CONFIG  |  DEBUG
+ *  FINE    |  DEBUG
+ *  FINER   |  TRACE
+ *  FINEST  |  TRACE
+ *  OFF     |  OFF
+ * </pre>
+ * <p>
+ * `FATAL` from previous bindings used by JMXFetch is now converted
+ * to `LogLevel.ERROR`.
+ * </p>
+ */
+public enum LogLevel {
+    ALL(0, "ALL"),
+    ERROR(1, "ERROR"),
+    WARN(2, "WARN"),
+    INFO(3, "INFO"),
+    DEBUG(4, "DEBUG"),
+    TRACE(5, "TRACE"),
+    OFF(6, "OFF");
```

Comment:
This order looks a bit weird to me. I would expect ALL to be number 6, and OFF number 0 to follow the order, but I might be missing something. Also I haven't checked where this order is used

Reply:
Addressed.